curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/65364566e84e378837fbaf2a.md

```diff
@@ -7,9 +7,10 @@ dashedName: step-33
 
 # --description--
 
-You need to hook up the `playSong` function to a `click` event listener so the songs will start playing when the play button is clicked.
 
-Add a click event listener to the `playButton` element, then use arrow syntax to pass in a callback with an empty pair of curly braces.
+In previous steps you built out the functionality for playing a song. Now you need to add the functionality to the play button so that it will play the current song when it is clicked on.
+
+Use the `addEventListener()` method and pass in a `click` event for the first argument and an empty callback function with arrow syntax for the second argument. Ex.(`()=>{}`). 
```

Comment:
```suggestion
Use the `addEventListener()` method and pass in the string `click` for the first argument as the event to listen for and an empty callback function with arrow syntax for the second argument. Ex.(`()=>{}`).
```
I think we should specify that `click` is a string that describes the type of event to listen for.

---

`click` is not an event, but a string and we can't use `"click"` because the style guide doesn't allow it (as far as I know).

Reply:
Before I make adjustments to this pull request, I just wanted to get some small clarification. When you say "click is not an event and we cant use `"click"` because the style guide does not allow it" I'm just a teeny bit lost hehe